content/en/docs/aws/rds.md

```diff
@@ -0,0 +1,92 @@
+done  
+```
+You can either use console or use attached CloudFormation template to deploy Amazon RDS database
+
+{{% alert title="Warning" color="warning" %}}
+The CloudFormation template deploys Amazon RDS for MySQL that is intended for Dev/Test environment.
+We highly recommend deploying Multi-AZ database for Production. Please review RDS [documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Welcome.html) to learn more
+{{% /alert %}}
+
+[{{<figure src="/docs/images/aws/cloudformation-launch-stack.png">}}](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=kubeflow-db&templateURL=https://cloudformation-kubeflow.s3-us-west-2.amazonaws.com/rds.yaml)
+
+Remember to select correct **region** for your cluster and click Next. We recommend you to change the **DBPassword**, if not it will dafault to `Kubefl0w`. Select VpcId, Subnets and SecurityGroupId before clicking Next. Take rest all defaults and click **Create Stack**. Once the CloudFormation is completed, click on Outputs tab to get RDS endpoint. We will use it in next step while installing Kubeflow   
+
+![dashboard](/docs/images/aws/cloudformation-rds-output.png)
+
+### Deploy Kubeflow Pipeline and Metadata using Amazon RDS
+
+1. Follow the [install documentation](https://www.kubeflow.org/docs/aws/deploy/install-kubeflow/) until [Deploy Kubeflow](https://www.kubeflow.org/docs/aws/deploy/install-kubeflow/#deploy-kubeflow) section.
+Modify `${CONFIG_FILE}` file to add `external-mysql` in both pipeline and metadata kustomizeConfigs and remove mysql database as shown below.  
+
+    ![dashboard](/docs/images/aws/external-mysql-rds.png)
+
+2. Run the follow commands to build Kubeflow installation:
+
+    ```
+    cd ${KF_DIR}
+    kfctl build -V -f ${CONFIG_FILE}
+    ```
+    This will create two folders `aws_config` and `kustomize` in your environment. Edit `params.env` file for the external-mysql pipeline service (`kustomize/api-service/overlays/external-mysql/params.env`) and update values based on your configuration
+
+    ```
+    mysqlHost=<$RDSEndpoint>
+    mysqlUser=<$DBUsername>
+    mysqlPassword=<$DBPassword>
+    ```
+    Edit `params.env` file for the external-mysql metedata service (`kustomize/metadata/overlays/external-mysql/params.env`) and update values based on your configuration
+
+    ```
+    MYSQL_HOST=external_host
+    MYSQL_DATABASE=<$RDSEndpoint>
+    MYSQL_PORT=3306
+    MYSQL_ALLOW_EMPTY_PASSWORD=true
+    ```
+    Edit `secrets.env` file for the external-mysql metedata service (`kustomize/metadata/overlays/external-mysql/secrets.env`) and update values based on your configuration
+
+    ```
+    MYSQL_USERNAME=<$DBUsername>
+    MYSQ_PASSWORD=<$DBPassword>
+    ```
+
+3. Run Kubeflow installation
+
+    ```
+    cd ${KF_DIR}
+    kfctl apply -V -f ${CONFIG_FILE}
+    ```
+Your pipeline and metadata is now using Amazon RDS.
```

Comment:
Can you  add s section to verify it's working? For example, query RDS and check database and tables there. Or a debug section.

Reply:
Thanks. I added troubleshooting rds connectivity issues in https://www.kubeflow.org/docs/aws/troubleshooting-aws/